rca/static_src/javascript/programmes/components/ProgrammesCategories/CategoriesPanels.js

```diff
@@ -18,7 +18,6 @@ const CategoriesPanels = ({ categories, activeCategory }) => {
                     key={c.id}
                     id={c.id}
                     className="categories-panels__panel"
-                    role="tabpanel"
```

Comment:
Fix this issue
https://app.circleci.com/pipelines/github/torchbox/rca-wagtail-2019/5435/workflows/12ed1ee6-7321-428d-a1ab-e110cf53753d/jobs/6993
```
/Users/bxu/Documents/rca-wagtail-2019/rca/static_src/javascript/programmes/components/ProgrammesCategories/CategoriesPanels.js
  17:17  error  The attribute aria-expanded is not supported by the role tabpanel  jsx-a11y/role-supports-aria-props
```

Reply:
take out role should have a minimum impact on the FE